svgnative/src/SVGStringParser.cpp

```diff
@@ -455,6 +456,7 @@ void ParsePathString(const std::string& pathString, Path& p)
                 currentY = startY;
             }
 
+            prev = *pos++;
```

Comment:
I don't see the issue right now. We have `*pos++` at the beginning of the for loop if `pos` is not a number, nor -, + or a dot. It isn't since we have `Z`. So `pos` is looking at the next char already (which might be a serious of wsp, the next path command or invalid). Why advancing `pos` here once again?

`prev` should be `Z` here already which is correct. Btw. `prev` has the last path segment type.

Reply:
Also, could you add the test that runs into an infinite loop to the unit testing as well please?